static/php/register.php

```diff
@@ -0,0 +1,80 @@
+<?php
+//get environment variables
+$envFile = __DIR__ . '/../env/.env';
+  if (file_exists($envFile)) {
+      $lines = file($envFile, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
+      foreach ($lines as $line) {
+          list($key, $value) = explode('=', $line, 2);
+          $_ENV[$key] = $value;
+          putenv("$key=$value");
+      }
+  }
+
+//set up variables with environment variables
+$DATABASE_HOST = getenv('DB_HOST');
```

Comment:
If Possible can we move this Into Separate Connection.php. and Then At The top include an Connection.php